sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/data-load.sh

```diff
@@ -14,36 +14,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#!/bin/sh
+#!/bin/bash -e
 
+recordcount=1000
 # Identify the pod
 cassandra_pods="kubectl get pods -l name=cassandra"
-running_seed="$(kubectl get pods -o json -l name=cassandra -o jsonpath='{.items[0].metadata.name}')"
-echo "$running_seed"
-cassandra_svc="kubectl get svc cassandra"
-echo
+running_seed="$(kubectl get pods -o json -l name=cassandra -o jsonpath='\
+{.items[0].metadata.name}')"
+echo "Detected Running Pod $running_seed"
+
+# After starting the service, it takes couple of minutes to generate the external IP for the 
+# service. Hence, wait for sometime.
 
-# After starting the service, it takes couple of minutes to generate the external IP for the service. Hence, wait for sometime.
 # Identify external IP of the pod
 external_ip="$(kubectl get svc cassandra -o jsonpath='{.status.loadBalancer.ingress[0].ip}')"
+echo "Waiting for the Cassandra service to come up ........"
 while [ -z "$external_ip" ]
 do
    sleep 10s
    external_ip="$(kubectl get svc cassandra -o jsonpath='{.status.loadBalancer.ingress[0].ip}')"
+   echo ".................."
```

Comment:
do you mean to output this many dots every time? I feel like just one is fine/standard.